posix.mak

```diff
@@ -113,16 +113,16 @@ endif
 DDOC_VARS_STABLE=\
 	  DOC_OUTPUT_DIR="${DOC_OUTPUT_DIR}/phobos" \
 	  STDDOC="$(addprefix $(PWD)/, $(STD_DDOC))" \
-	  DMD="$(DMD_STABLE)" \
-	  DRUNTIME_PATH="${DRUNTIME_DIR}" \
+	  DMD="$(realpath $(DMD_STABLE))" \
```

Comment:
Please no `realpath`, it resolves symlinks, and the final paths (which may be temporary and expose implementation details) end up all over the place, sometimes even in outputs.